package.json

```diff
@@ -3,29 +3,46 @@
   "version": "2.0.0-beta3",
   "description": "SignalFx library to collect nodejs metrics and report to SignalFx",
   "main": "src/index.js",
+  "engines": {
+    "node": ">=8.0.0 <12",
+    "npm": ">=5.0.0 <7"
+  },
   "scripts": {
+    "build": "node-pre-gyp build",
+    "install": "node-pre-gyp install --fallback-to-build",
+    "lint:fix": "eslint src --fix",
     "lint": "eslint src",
-    "test": "mocha"
+    "pretest": "npm run build",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "binary": {
```

Comment:
What are those for? Distribution?

Reply:
`node-pre-gyp` requires these and I want to make as few changes as possible